daml/Marketplace/Exchange.daml

```diff
@@ -132,9 +136,15 @@ template Exchange
             $ minQuantity < maxQuantity
 
           assertMsg "Pair must have different base and quote tokens" $ baseToken.id.label /= quoteToken.id.label
-          let instrumentName = baseToken.id.label <> quoteToken.id.label
+
+          let instrumentName = if clearedMarket
+              -- TODO : figure out how to pad instrument names to a length of 13, then append CLR or 000
+              then baseToken.id.label <> quoteToken.id.label <> "CLR"
```

Comment:
my haskell-foo is not strong enough to write a function that takes a shorter input string and a max length, and returns a padded string with as many instances of `0` to the end of the input string to fill up to the max length...

Reply:
may be a nicer way but:
```
addZeros : Int -> Text -> Text
addZeros n txt = T.implode $ T.explode txt <> replicate (n - T.length txt) "0"
```

Reply:
Gonna leave this as a simple "CLR" appending for now, and I'll look into creating unique instrument IDs this week / next week